samples/nrf_desktop/src/modules/ble_scan.c

```diff
@@ -95,6 +144,9 @@ static void scan_connecting(struct bt_scan_device_info *device_info,
 	LOG_INF("Connecting done");
 
 	bt_addr_le_copy(&target_addr, device_info->addr);
+
+	settings_save_one(MODULE_NAME "/taddr", &target_addr,
```

Comment:
Maybe target address should be saved after pairing, not after connecting? What if you connect, but not be able to pair with peripheral for some reason?

Reply:
I'm ok with both ways, as long as reconnecting works reasonably well.